src/components/AmountFilter.jsx

```diff
@@ -74,7 +70,7 @@ export default function AmountFilter(props) {
 }
 
 AmountFilter.propTypes = {
-  minAmount: PropTypes.number.isRequired,
-  maxAmount: PropTypes.number.isRequired,
-  onAmountFilterChange: PropTypes.func.isRequired,
+  min: PropTypes.number.isRequired,
+  max: PropTypes.number.isRequired,
+  setQueryParam: PropTypes.func.isRequired,
```

Comment:
Hmm, I recommend having separate `onMaxChange` and `onMinChange` functions. It's a bit messy to have a specific component like this understand query parameters.

Reply:
Unless this component has some sort of constant that gets used elsewhere, e.g. `QUERY_PARAM_MIN_AMOUNT='minAmount'`

Reply:
Just added some notes at the end of the guide:
https://github.com/beyondhb1079/s4us/wiki/Guide:-Adding-URL-parameter-support

It's subjective whether `AmountFilter` should be aware of query params or not the way `FilterBar` is. Personally I think `onMaxChange` and `onMinChange` functions probably keep this component slightly cleaner.